smashed/mappers/tokenize.py

```diff
@@ -162,3 +162,51 @@ def _transform(tokens: List[str]) -> List[str]:
         # new_data = {f'{self.prefix}{k}': v if k not in self.input_fields
         #             else _transform(v) for k, v in data.items()}
         # return new_data
+
+
+class PaddingMapper(SingleBaseMapper):
```

Comment:
If you want to pad multiple fields w/ different sybols, would you run many `PaddingMapper` one after the other? I can see that being a bit inefficient. 

Proposal: have `PaddingMapper` take a `fields_pad_ids: Dict[str, int]` as input, and use that to pad multiple fields at once.